packages/application-shell/src/components/version-tracker/version-tracker.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import flowRight from 'lodash/flowRight';
+import { connect } from 'react-redux';
+import { withApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import applicationShellVersion from '../../version';
+import pkg from '../../../package.json';
```

Comment:
I would not import it like this. For example, if someone were to use CRA, it would not work because of imports outside of `src`.

Since we know then name of the package, I would just hardcore it

Reply:
Sure.